packages/components/src/navigation/styles/navigation-styles.js

```diff
@@ -50,12 +50,14 @@ export const MenuBackButtonUI = styled( Button )`
 export const MenuTitleUI = styled( Text )`
 	padding: 4px 0 4px 16px;
 	margin-bottom: 8px;
+	color: #f0f0f0;
```

Comment:
Could we use `G2.gray[100]` instead of hardcoding the hex value?

Reply:
Replaced it with G2